.github/workflows/salesloft.yml

```diff
@@ -0,0 +1,75 @@
+name: SalesLoft
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+jobs:
+  rspec:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - rails: "7.0"
+            ruby: 3.2
+          - rails: 6.1
+            ruby: 2.7
+          - rails: 6.1
+            ruby: 3.2
+
+          - rails: 5.2
+            ruby: 2.7
+
+
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: postgresql://postgres:postgres@localhost/postgres
```

Comment:
You can remove this database_url env var because no database for this gem

Reply:
You sure about that 👀

Reply:
😂 🤷 I guess not?

Reply:
I guess it will fail if it's removed if it's needed....

Reply:
I was just messing with you. This repo is still sending me updates for some reason, and I saw your comment come across.

Reply:
I say keep it. You can never be too cautious.